Neos.Flow/Configuration/Settings.Object.yaml

```diff
@@ -51,3 +51,13 @@ Neos:
         #  - '^Neos\\SomePackage\\ValueObjects\\SomeSpecificValueObject$'
         #
         excludeClassesFromConstructorAutowiring: []
+
+        # This is a forward compatibility setting, preventing constructor arguments to be autowired
+        # if they are not declared singleton or session when this setting is false.
+        #
+        # BEWARE of dragons, Flow code should be fine, but Neos might still contain such constructs.
+        # Manual full cache flush is necessary after changing this setting.
+        #
+        # In the future autowiring of prototypes should no longer be an option as it makes no sense.
+        # Use a factory class if you need this otherwise.
+        prototypeAutowiring: true
```

Comment:
The name is not as clear to me, as it should be. Does this prevent the autowiring of constructor arguments _to_ prototypes or does it prevent the use of prototypes _as_ constructor arguments?

Reply:
fixed via https://github.com/neos/flow-development-collection/pull/3494/commits/7b363e2a3d71df17165cd68a5bcc80c8a6175717